client/src/Admin/Hospitals.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext, useState, useEffect} from 'react';
+import Context from '../Context';
+import ApiService from '../ApiService';
+
+function Hospitals() {
+  const { organization} = useContext(Context);
+  const [hospitals, setHospitals] = useState([]);
+
+  useEffect(() => {
+      ApiService.organizations.get(organization.id).then((response) => {
+        if (organization?.type === 'HEALTHCARE') {
+          setHospitals(response.data.hospitals);
+        } 
+      });
+    }, [organization]);
+
+  return (
+    <>
+      {organization.type === "HEALTHCARE" &&
```

Comment:
Created a rudimentary list of hospitals. I'm thinking we can better style the list of hospitals in a separate PR when that's needed